content/en/blog/_posts/2023-11-14-PodReadyToStartContainersCondition-in-beta.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog
+title: "Kubernetes 1.29: PodReadyToStartContainersCondition in beta"
+date: 2023-11-13
+slug: pod-ready-to-start-containers-condition-now-in-beta
+---
+
+**Author**: Kevin Hannon(Red Hat)  zefeng chen
+
+
+With Kubernetes 1.29 the PodReadyToStartContainersCondition functionality entered beta and is available by default.
+A new kubelet-managed pod condition in the status field of a pod. The kubelet will use the PodHasNetwork condition to accurately surface the initialization state of a pod,
```

Comment:
```suggestion
A new kubelet-managed pod condition in the status field of a pod. The kubelet will use the PodReadyToStartContainers condition to accurately surface the initialization state of a pod,
```

Reply:
Style nit: condition names should be inside backticks. This helps people tell them apart from API kinds (which we write without backticks).

Reply:
updated Kevin's comment. 

> Style nit: condition names should be inside backticks. This helps people tell them apart from API kinds (which we write without backticks).

can you list me an example? I'm not quite getting this. sorry for my poor English